### PR TITLE
Feature: unify projects

### PR DESCRIPTION
**Pull request by rainbowdashlabs**
**Checklist**
- [ ] I made changes to commands
  - [ ] I fully localised the command
  - [ ] I fully checked the commands functionality

- [ ] I added new localisation codes
  - [ ] I fully translated it for all languages
  - [ ] I sorted properties alphabetically

- [ ] I made changes to the database
  - [ ] I added my changed to a patch file
  - [ ] I made sure my database was up to date
  - [ ] I checked that the migration works

- [x] I made changes to internal structure 

**Short Description**
- Add license header
- Switch to template renovate configuration
- Rename development to dev
- Rename master to main
- Adjust workflow files for new names
- Use version catalogue to group dependencies instead of renovate
- Switch to log utils lib instead of internal log appender

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   push:
     tags:
     branches:
-      - master
+      - main
       - development
```
Comment (Taucher2003) [inline .github/workflows/docker.yml:8]:
```suggestion
      - dev
```

File: settings.gradle.kts [hunk 2]
```diff
@@ -3,3 +3,29 @@ rootProject.name = "rep-bot"
 plugins {
     id("org.gradle.toolchains.foojay-resolver-convention") version("0.6.0")
 }
+
+dependencyResolutionManagement {
+    versionCatalogs {
```
Comment (Taucher2003) [inline settings.gradle.kts:8]: I'm not sure, is renovate able to handle this?

File: settings.gradle.kts [hunk 3]
```diff
@@ -3,3 +3,29 @@ rootProject.name = "rep-bot"
...
+        create("libs") {
+            // misc
+            library("jetbrains-annotations", "org.jetbrains:annotations:24.0.1")
+            version("sadu", "1.3.1-DEV")
```
Comment (Taucher2003) [inline settings.gradle.kts:12]: We are going to a dev version again?

File: settings.gradle.kts (hunk 2 above)
Comment (rainbowdashlabs) [reply settings.gradle.kts:8]: Yes it is and that is the nice stuff! xD

File: settings.gradle.kts (hunk 3 above)
Comment (rainbowdashlabs) [reply settings.gradle.kts:12]: Actually I think we dont need to... We dont use the feature which is fixed there.

File: settings.gradle.kts (hunk 2 above)
Comment (rainbowdashlabs) [reply settings.gradle.kts:8]: Renovate even recognises the shared versions and groups them into a single update pr

Comment (rainbowdashlabs) [reply settings.gradle.kts:8]: Can be seen here: https://github.com/rainbowdashlabs/krile/issues/5

File: .github/workflows/docker.yml (hunk 1 above)
Comment (Taucher2003) [reply .github/workflows/docker.yml:8]: This is still `development` but the branch is named `dev`.

**Review by Taucher2003 — APPROVED** [review]

**Pull request merged**